.github/workflows/windows.yml

```diff
@@ -0,0 +1,84 @@
+name: Build and Package FLINT for Windows
+
+on:
+  push:
+    branches:
+      - Windows_workflow
+  pull_request:
+    branches:
+      - Windows_workflow
+
+jobs:
+  build-and-package:
+    name: Build and Package FLINT for Windows
+    runs-on: windows-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: Windows_workflow  # Checkout the Windows_workflow branch
+
+      - name: Install CMake
+        run: |
+          $cmakeInstaller = "cmake-3.29.6-windows-x86_64.msi"
+          Invoke-WebRequest -Uri https://github.com/Kitware/CMake/releases/download/v3.29.6/$cmakeInstaller -OutFile $cmakeInstaller
+          Start-Process msiexec.exe -ArgumentList "/i", $cmakeInstaller, "/quiet", "/qn", "/norestart", "ADD_CMAKE_TO_PATH=System" -NoNewWindow -Wait
+
+      - name: Install Git
+        run: |
+          $gitInstaller = "Git-2.41.0-64-bit.exe"
+          Invoke-WebRequest -Uri https://github.com/git-for-windows/git/releases/download/v2.41.0.windows.1/$gitInstaller -OutFile $gitInstaller
+          Start-Process $gitInstaller -ArgumentList "/VERYSILENT", "/NORESTART" -NoNewWindow -Wait
+
+      - name: Install Visual Studio Build Tools
+        run: |
+          $vsInstaller = "vs_buildtools.exe"
+          Invoke-WebRequest -Uri https://aka.ms/vs/16/release/vs_buildtools.exe -OutFile $vsInstaller
+          Start-Process $vsInstaller -ArgumentList "--quiet", "--wait", "--norestart", "--nocache", "--installPath", "C:\BuildTools", "--add", "Microsoft.VisualStudio.Workload.VCTools;includeRecommended" -NoNewWindow -Wait
+
+      - name: Install Boost libraries
```

Comment:
hmm i also noticed @mfellows has this updated script that installs dependencies and builds flint. I think we can call the script in the workflow